.github/workflows/npmpublish.yml

```diff
@@ -36,9 +36,15 @@ jobs:
           registry-url: https://registry.npmjs.org/
 
       - name: Install
+        run: npm ci
+
+      - name: Build
+        run: npm run build:prod
+
+      - name: Install Production
         run: npm ci --production
 
       - name: Publish
-        run: npm publish
+        run: npm publish --prepublishOnly
```

Comment:
This CLI flag doesn't exist :thinking:  
I think you misunderstood it
Cause this is already merged I will open a new PR to fix that

Reply:
Of course it exists ;)
It even works.

Reply:
prepublishOnly is a lifecycle script -> https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts

Please link a documentation about you are talking
I don't know the CLI flag you are talking about